requirements-test.txt

```diff
@@ -1,2 +1,4 @@
+coverage==4.5.3
```

Comment:
4.5.3 looks quite old. Why not 5.5?

Reply:
Hmm, this version was installed on my machine. updated to 5.*